apps/authentication/confirm/password.py

```diff
@@ -2,11 +2,12 @@
 
 from authentication.mixins import authenticate
 from .base import BaseConfirm
+from ..const import ConfirmType
 
 
 class ConfirmPassword(BaseConfirm):
-    name = 'password'
-    display_name = _('Password')
+    name = ConfirmType.PASSWORD.value
+    display_name = ConfirmType.PASSWORD.name
 
     def check(self):
         return self.user.is_password_authenticate()
```

Comment:
There is no error in the code snippet provided and there might not be any need to apply any optimizations. The current implementation seems correct.